Annotation/constituent_golden_set.json

```diff
@@ -606,8 +685,21 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
-                "embedded clauses": []
+                "clause": "how real and honest they are about how they could forgive",
+                "embedded clauses": [
+                    {
+                        "predicate": [
+                            {
+                                "str": "are about",

```

Comment:
I agree this is a constituent embedded clause but the real and honest are adjectives part of the predicate I think (be real and honest about...) so maybe it's actually 2 predicates (one for each adjective)?

Reply:
yeh potentially, I think this is gonna be a real tricky case that our heuristics aren't going to be able to capture? I'm going to leave this one for now so we can discuss later